test/email_reply_parser_test.rb

```diff
@@ -20,6 +21,36 @@ def test_encoding_should_be_maintained
     end
   end
 
+  def test_language(language_abbreviation)
+    # Get the number of files in the given language directory
+    file_count = Dir[EMAIL_FIXTURE_PATH.join("#{language_abbreviation}/*")].size
+
+    file_count.times do |current_file_index|
+      # Files are named starting with index 1 so begin iterator at 1
+      file_name = current_file_index + 1
+
+      # Read the body of the email for each of the files in the iteration and convert to a string
+      body = IO.read EMAIL_FIXTURE_PATH.join("#{language_abbreviation}/#{file_name}.txt").to_s
+      reply = EmailReplyParser.read(body)
+      assert_equal(base_email_content(language_abbreviation), reply.visible_text)
+    end
+  end
+
+  def test_fr_language
+    test_language(:fr)
+  end
+  
+  def test_nl_language
+    test_language(:nl)
+  end
+
+  def test_en_language
+    test_language(:en)
+  end
+  def test_it_language
```

Comment:
Can you add a line between these two method definitions?

Reply:
Done!